credentialProvider/src/test/java/org/frankframework/credentialprovider/CredentialResolverCredentialFactoryTest.java

```diff
@@ -0,0 +1,60 @@
+package org.frankframework.credentialprovider;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.io.InputStream;
+import java.net.URL;
+import java.util.Properties;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import org.frankframework.credentialprovider.util.CredentialConstants;
+import org.frankframework.util.StringResolver;
+
+/**
+ * Test the working of resolving of credentials with and without a configured credential factory
+ */
+public class CredentialResolverCredentialFactoryTest {
+
+	Properties properties;
+
+	@BeforeEach
+	void setUp() throws Exception {
+		properties = new Properties();
+		URL propertiesURL = getClass().getResource("/StringResolver.properties");
+		assertNotNull(propertiesURL, "properties file [StringResolver.properties] not found!");
+
+		InputStream propsStream = propertiesURL.openStream();
+		properties.load(propsStream);
+		assertFalse(properties.isEmpty(), "did not find any properties!");
+
+		System.setProperty("authAliases.expansion.allowed", "${allowedAliases}");
+
+		CredentialFactory.clearInstance();
```

Comment:
Het lijkt me een goed idee om ook in de `@AfterEach` de credential factory weer op te schonen, en de property `authAliases.expansion.allowed` terug te zetten op de oorspronkelijke waarde.